templates/login.html

```diff
@@ -7,16 +7,16 @@
 <div class="w-full max-w-md text-center mx-auto">
   <div class="bg-white shadow-md rounded px-8 pt-6 pb-4 mb-4">
       <form class="js-validated-form auth" style="border: none;" onsubmit="event.preventDefault (); hedyApp.auth.submit ('login')">
-        <h2 class="py-2">{{auth.login_long}}</h2>
+        <h2 class="py-2">{{ _('Log in to your account') }}</h2>
         <div class="mb-4">
-            <label for="username">{{auth.username}}</label>
+            <label for="username">{{ _('Username') }}</label>
             <input id="username" name="username" required>
         </div>
         <div class="mb-4">
-            <label for=password">{{auth.password}}</label>
+            <label for=password">{{ _('Password') }}</label>
             <input id="password" name="password" type=password required>
         </div>
-        <button type="submit" class="green-btn mt-2">{{auth.login}}</button>
+        <button type="submit" class="green-btn mt-2">{{ _('Log in') }}</button>
```

Comment:
You can create different domains or different context for strings. The code surrounding te "log in" text might give the translator a clue, otherwise screenshots or comments can be added. Special comments in code are transferred to translations files as well, so coders can help translators when needed.

Reply:
What I mean is that in both cases the input would be the same, but the output needs to be different. How is that distinction made?

Reply:
Multiple (the same) input strings are possible if they have a different context.

Reply:
To clarify, then one contact would be "infinitive" and one would be "imperative"? That seems to stretch the limit of what a context is a bit?

Reply:
For now I copied the yaml keys as we decided to stay away for default English and use keys instead. So the current solution is "login_long" and "login".

Reply:
Sounds good although maybe something like `login_imperative` and `login_infinitive` might be more descriptive (and more helpful to translators)?